queries/c/highlights.scm

```diff
@@ -142,6 +141,10 @@
 (type_definition
   declarator: (type_identifier) @type.definition)
 
+(primitive_type) @type.builtin
```

Comment:
Primitive types not just simple 'type', with 'builtin' hl the color schemes can differentiate that see attached image blue return types in function.